packages/api/src/router/exports.ts

```diff
@@ -256,28 +239,16 @@ export const exportsRouter = router({
     )
     .mutation(async ({ input, ctx }) => {
       try {
-        const userId = ctx.auth.userId;
-        const { chatId, data, messageId } = input;
-        const lessonSnapshots = new LessonSnapshots(ctx.prisma);
-        const { id: snapshotId } = await lessonSnapshots.getOrSaveSnapshot({
-          userId,
-          chatId,
-          snapshot: data,
-          trigger: "EXPORT_BY_USER",
-          messageId,
-        });
         // find the latest export for this snapshot
-        const exportData = await ailaGetExportBySnapshotId({
-          prisma: ctx.prisma,
-          snapshotId,
-          exportType: "LESSON_SLIDES_SLIDES",
+        const exportType = "LESSON_SLIDES_SLIDES";
```

Comment:
```
enum LessonExportType {
  STARTER_QUIZ_DOC
  EXIT_QUIZ_DOC
  LESSON_PLAN_DOC
  LESSON_SLIDES_SLIDES
  WORKSHEET_SLIDES
  ADDITIONAL_MATERIALS_DOCS

  @@schema("public")
}
```

Seems like it is the correct name

Reply:
I'd expect it to be LESSON_PLAN_SLIDES rather than LESSON_SLIDES_SLIDES - reads like a typo. Not suggesting this is a new code issue, but presumably this is hard to change if it's a database enum

Reply:
Yeah for sure, was surprised when i saw it